test/fixtures/track-long-names.json

```diff
@@ -40,6 +40,7 @@
       {
         "_eventName": "Testing in case someone sends us events ",
```

Comment:
Oh, I see. Oddly enough, it is because if you enter in a long event name, there's an if statement in our code that causes eventName to be read from msg.event(), instead of appEvents[eventName] - which circumvents the bug.